.github/workflows/deb/debian/compat

```diff
@@ -0,0 +1 @@
+11
```

Comment:
I know you know these commits should be part of the pull request, since they are marked [REMOVE], but I am wondering, why include them in the first place?

Reply:
Ah, sorry. This was cherry-picked en masse from #280 to show that this builds and runs fine on as many distributions as possible. It's just a copy of the commit there, I'll remove it completely.

Tests (these are not testing the new functionality anymore, though): [one](https://github.com/hg/tinc/actions/runs/1049291632), [two](https://github.com/hg/tinc/actions/runs/1049305258), [three](https://github.com/hg/tinc/actions/runs/1049321844)